app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+         a dialog box shown to the user when a settings change requires the application to restart. -->
     <string name="restart_dialog_restart">Restart Required</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_text): The string 'restart_dialog_text' is the text displayed
+         in the restart dialog box explaining why the restart is required. -->
     <string name="restart_dialog_text">You must restart the Firefox Reality app in order to complete changes. Would you like to do that now?</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_button):  The string 'restart_dialog_button' is used to
+         label the button the user presses to restart the application in the restart dialog. -->
     <string name="restart_dialog_button">Restart Now</string>
+    <!-- LOCALIZATION NOTE (developer_options_remote_debugging): The string 'developer_options_remote_debugging'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle remote debugging
+         of web content in the application. -->
     <string name="developer_options_remote_debugging">Enable Remote Debugging</string>
+    <!-- LOCALIZATION NOTE (developer_options_show_console): The string 'developer_options_show_console'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle redirecting
+         console output to the Android logcat. -->
     <string name="developer_options_show_console">Redirect Console to Logcat</string>
+    <!-- LOCALIZATION NOTE (developer_options_env_override): The string 'developer_options_env_override'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle loading externally
+         generated environments and overriding the ones built in. -->
     <string name="developer_options_env_override">Enable Environment Override</string>
+    <!-- LOCALIZATION NOTE (developer_options_multiprocess): The string 'developer_options_multiprocess'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle running content
+         in a separate process. -->
     <string name="developer_options_multiprocess">Enable Multiprocess</string>
+    <!-- LOCALIZATION NOTE (developer_options_ua_mode): The string 'developer_options_ua_mode' is
+         used to label a set of radio buttons that allow the user to change the user agent (UA) string
+         of the browser. -->
     <string name="developer_options_ua_mode">User Agent Mode</string>
+    <!-- LOCALIZATION NOTE (developer_options_events): FIXME This option should be removed -->
     <string name="developer_options_events">Input Events</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_density">Display Density:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_window_size">Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
```

Comment:
these need context

Reply:
This was a wip patch to make sure I was on the right track. I'll try and finish up the first cut today (writing the comments is quite tedious).